src/main/java/edu/cornell/kfs/module/purap/jaggaer/contract/xml/Authentication.java

```diff
@@ -0,0 +1,39 @@
+package edu.cornell.kfs.module.purap.jaggaer.contract.xml;
+
+import jakarta.xml.bind.annotation.XmlAccessType;
+import jakarta.xml.bind.annotation.XmlAccessorType;
+import jakarta.xml.bind.annotation.XmlElement;
+import jakarta.xml.bind.annotation.XmlRootElement;
+import jakarta.xml.bind.annotation.XmlType;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "identity",
+    "sharedSecret"
+})
+@XmlRootElement(name = "Authentication")
+public class Authentication {
```

Comment:
I think this is a duplicate of edu.cornell.kfs.module.purap.jaggaer.supplier.xml.Authentication. Can contacts use the already existing object?  If so, maybe the existing one should be moved to a different package indicating it is used by multiple Jaggaer integrations

Reply:
I'm not sure whether it's safe to consolidate the two Authentication classes entirely, in case Jaggaer makes future changes to their authentication setup. However, since they currently have the same content, I could add a base class to hold the common portions and then update the supplier/contract variants to extend from the base class. That will make it easier to separate the two if Jaggaer ends up making changes to one API but not the other.